src/content/contributing/translation-program/blog-translations/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: EF blog translations
+lang: en
+description: How to contribute by translating the Ethereum Foundation blog
+sidebar: true
+---
+
+# EF blog translations {#blog-translations}
+
+The [Ethereum Foundation blog](https://blog.ethereum.org/) is the primary publication tool for a lot of EF-supported teams. Several teams regularly use the blog to publish announcements, updates, reports, roundups, and other public communication.
+
+In line with some key values from the [ethereum.org Translation Program mission and vision](/contributing/translation-program/mission-and-vision) - accessibility, inclusivity, empowerment, and building the global Ethereum community - we want to translate the Ethereum Foundation blog posts in as many languages as possible.
+
+The blog translation project is open and anyone can contribute.
+If you want to get involved and help translate blog posts into your language, read on to learn how to join the project in Crowdin and start translating.
+
+## How to translate {#how-to-translate}
+
+<InfoBanner shouldCenter emoji=":light_bulb:">
+  We encourage you to check out the ethereum.org Translation style guide. It contains some of the most important guidelines, instructions, and tips for translators to help localize the blog. While originally written for ethereum.org translators, most of the style guide is relevant to blog translations as well.
+  {" "}<Link to="/contributing/translation-program/translators-guide/">View the Translation Style Guide</Link>
+</InfoBanner>
+
+1. **[Join the blog translations project on Crowdin](https://crowdin.com/project/ethereum-foundation-blog)**  
+   You will need to log in to your Crowdin account or sign up if you don’t already have one. All that is required to sign up is an e-mail account and password.
+
+2. **Open the language you want to translate and find a blog post to work on**  
```

Comment:
I suspect we should include an answer to "what if I want to translate a blog post but don't see it in Crowdin?", e.g. providing instructions to contact our team & ask us to add the blog post file to Crowdin.

Reply:
On this front, we should probably explain our general approach, e.g. that we'll focus on new posts moving forward (or however we intend to prioritize posts to be translated).

Reply:
**Open the language you want to translate and find a blog post to work on**

This is two steps

- Open the language you want to translate
- find a blog post to work on